index.html

```diff
@@ -4481,7 +4502,61 @@ <h2><span class="chunk">cICP</span>
 [[ITU-T H.273]] and
 it's associated recommendations.</p>
 </section>
-</section>
+
+<!-- Maintain a fragment named "11iCCN" to preserve incoming links to it -->
+<section id="11iCCN">
+  <h2><span class="chunk">iCCN</span>
+    Embedded ICC Profiles v4 and UTF-8 Profile Names</h2>
+  
+    <p>iCCN profiles are the same is iCCP chunks but can support UTF-8 Profile
+      Names</p>
+    
+    <p>The four decimal values below correspond to the four-byte iCCN chunk type field:</p>
+    
+    <pre>
+    105 67 67 78
+    </pre>
+    
+    <p>The <span class="chunk">iCCN</span> chunk contains:</p>
+    
+    <table class="Regular" summary=
+    "This table defines the iCCP chunk">
+    <tr>
+    <td class="Regular">Profile name</td>
+    <td class="Regular">UTF-8 (character string)</td>
+    </tr>
+    
+    <tr>
+    <td class="Regular">Null separator</td>
+    <td class="Regular">1 byte (null character)</td>
+    </tr>
+    
+    <tr>
+    <td class="Regular">Compression method</td>
+    <td class="Regular">1 byte</td>
+    </tr>
+    
+    <tr>
+    <td class="Regular">Compressed profile</td>
+    <td class="Regular">n bytes</td>
+    </tr>
+    </table>
+    
+    <p>The iCCN profile name may be any convenient name for referring to
+    the profile. It is case-sensitive. Profile names shall contain printable 
+    Latin-1 characters, spaces and includes UTF-8 characters.  Leading, trailing,
+    and consecutive spaces are not permitted.</p>
+  
+    <p>PNG files should not simultaneously contain embedded iCCP and iCCN tags.</p>
```

Comment:
Is that still true? It is now possible to put an ICC.2 (iccMax) profile into iCCN and an older ICC.1 profile into iCCP. So I suggest dropping this paragraph.